File: SingleAPIExample/CSharp/VerifyPayment.cs
using Splitit.SDK.Client.Api;
using Splitit.SDK.Client.Model;

namespace Examples
{
    public class VerifyPaymentExample
    {
        InstallmentPlanApi PlanApi { get; }

        public void VerifyPayment(string installmentPlanNumber)
        {
            var verifyResponse = PlanApi.InstallmentPlanVerifyPayment(new VerifyPaymentRequest(installmentPlanNumber));
        }
    }
}

File: SingleAPIExample/PHP/VerifyPayment.php
<?php

$sessionId = '{{SessionID}}';
\SplititSdkClient\Configuration::sandbox()->setApiKey('{{APIKey}}');

$apiInstance = new SplititSdkClient\Api\InstallmentPlanApi(\SplititSdkClient\Configuration::sandbox(), $sessionId);

$request = new \SplititSdkClient\Model\VerifyPaymentRequest([
    'installment_plan_number' => '{{InstallmentPlanNumber}}',
]);

try {
    $result = $apiInstance->installmentPlanVerifyPayment($request);
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when calling InstallmentPlanApi->installmentPlanVerifyPayment: ', $e->getMessage(), PHP_EOL;
}
